Polyhedron/demo/Polyhedron/Plugins/PMP/Selection_plugin.cpp

```diff
@@ -285,15 +299,18 @@ public Q_SLOTS:
   }
   // Create selection item for selected polyhedron item
   void on_Create_selection_item_button_clicked() {
-    Scene_polyhedron_item* poly_item = getSelectedItem<Scene_polyhedron_item>();
+    std::cerr << "A"<< std::endl;
+    Scene_face_graph_item* poly_item = getSelectedItem<Scene_face_graph_item>();
     if(!poly_item) {
       print_message("Error: there is no selected polyhedron item!");
       return; 
     }
     // all other arrangements (putting inside selection_item_map), setting names etc,
     // other params (e.g. k_ring) will be set inside new_item_created
+    std::cerr << "B"<< std::endl;
     Scene_polyhedron_selection_item* new_item = new Scene_polyhedron_selection_item(poly_item, mw);
     ui_widget.modeBox->setCurrentIndex(last_mode);
+    std::cerr << "C"<< std::endl;
```

Comment:
oops. I just spoted the std::cerr << "A"  etc.  I think I accidentally committed this debug output.